test/test_grid_file.py

```diff
@@ -238,7 +241,7 @@ def test_grid_out_cursor_options(self):
         cursor_dict.pop('_Cursor__session')
         cursor_clone_dict = cursor_clone.__dict__.copy()
         cursor_clone_dict.pop('_Cursor__session')
-        self.assertDictEqual(cursor_dict, cursor_clone_dict)
+        self.assertEqual(cursor_dict, cursor_clone_dict)
```

Comment:
Explanation? Mypy should understand that `cursor.__dict__.copy()` is a dict, no?

Reply:
Ah, this is left over from earlier work, reverting.